### PR TITLE
Fix incorrect parsing of token ID in some NFTokenMint transactions

### PR DESCRIPTION
**Pull request by ledhed2222**
## High Level Overview of Change

In some cases, when parsing an NFTokenMint transaction for its created token ID, we were incorrectly determining this value. This would happen in cases where the mint transaction resulted in a single page being split into two, instead of when a single page had the new token appended, or when a single page was created to hold the sole token.

### Context of Change

See above

### Type of Change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### TypeScript/Hooks Update

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

## Before / After

If you examine transaction `C4E598099A8B13C5C8D2B8C86385A37B64C2F62BFA1FB87196401BB6ACB67A69` in the NFT sandbox, you'll see that the parsed token ID is `0008000085D33F9C5481D3515029C9904D16F0109414D3A00000099A00000000`. This is incorrect; it should be `0008000085D33F9C5481D3515029C9904D16F0109414D3A0DCBA29BA00000020`.

## Test Plan

I created an account on the sandbox and minted 33 tokens for them. The final mint transaction previously would report the wrong token ID. Now it reports the correct token ID.

### CODE REVIEW

**Comment by ckniffen** [comment]
Can you add a test case with that example transaction?

**Comment by ckniffen** [comment]
package-lock.json should not have changed.

File: .gitignore [hunk 1]
```diff
@@ -16,6 +16,7 @@ src/**/*.css.map
 .env.test.local
 .env.production.local
 .env
+.node-version
```
Comment (ckniffen) [inline .gitignore:19]: Files like `.node-version` or `.nvmrc` should be commited to a repo.

**Comment by ledhed2222** [comment]
> package-lock.json should not have changed.

yeah - it looks like it's because i'm using npm 8 instead of npm 7, which is allowed by our config. there's only a one line change in the package-lock so i think it's correct

**Review by ckniffen — APPROVED** [review]

**Review by shawnxie999 — APPROVED** [review]

**Pull request merged**